packages/jest-worker/src/types.ts

```diff
@@ -202,38 +201,27 @@ export type OnStateChangeHandler = (
 
 // Messages passed from the parent to the children.
 
-export type MessagePort = typeof EventEmitter & {
-  postMessage(message: unknown): void;
-};
-
-export type MessageChannel = {
-  port1: MessagePort;
-  port2: MessagePort;
-};
-
 export type ChildMessageInitialize = [
-  typeof CHILD_MESSAGE_INITIALIZE, // type
-  boolean, // processed
-  string, // file
-  Array<unknown> | undefined, // setupArgs
-  MessagePort | undefined, // MessagePort
+  type: typeof CHILD_MESSAGE_INITIALIZE,
+  isProcessed: boolean,
+  fileName: string,
+  setupArgs: Array<unknown>,
```

Comment:
Also it looks safe to remove `undefined`: https://github.com/facebook/jest/blob/c791d97114024391b1894d9923688c0a0216cc4f/packages/jest-worker/src/types.ts#L163